apidoc/Titanium/Media/Media.yml

```diff
@@ -1361,30 +1414,68 @@ properties:
     permission: read-only
 
   - name: VIDEO_SCALING_MODE_FILL
+    deprecated:
+        since: '7.0.0'
+        notes: This property has been removed for iOS in Titanium SDK 7.0.0 as of the official deprecation by Apple.
+        removed: '7.0.0'
     summary: Video is scaled until both dimensions fit the screen exactly, stretching if necessary.
     description: |
         The aspect ratio of the video is not preserved.
     type: Number
     permission: read-only
     platforms: [android, iphone, ipad]
   - name: VIDEO_SCALING_NONE
+    deprecated:
+        since: '7.0.0'
+        notes: This property has been removed for iOS in Titanium SDK 7.0.0 as of the official deprecation by Apple.
+        removed: '7.0.0'
     summary: Video scaling is disabled.
     type: Number
     permission: read-only
+  - name: VIDEO_SCALING_RESIZE
+    summary: Specifies that the video should be stretched to fill the layer’s bounds.
```

Comment:
That `’` probably fails validation.

Reply:
At other places in doc also  '  is used. So I think no problem with this.